src/Mantasflowers.Contracts/Review/GetProductReviewsResponse.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Mantasflowers.Contracts.Review
+{
+    public class GetProductReviewsResponse
+    {
+        public Guid ProductId { get; set; }
+
+        public Guid UserId { get; set; }
```

Comment:
There's no need to send user id back. Also, noticed there's a review (submission/creation) date missing.

Reply:
I mapped the date from `UpdateOn`.